server/Alumni/Controllers/UserController.cs

```diff
@@ -0,0 +1,58 @@
+using Microsoft.AspNetCore.Mvc;
+using Alumni.Models;
+
+
+namespace Alumni.Controllers;
+
+[ApiController]
+[Route("api/users")]
```

Comment:
I want to use 
- GET - `api/users` get all users (REST)
- POST - `api/user` create new user
when add a 'controller level route' it decreases naming flexibility

Reply:
done